wolfcrypt/src/cmac.c

```diff
@@ -328,5 +342,26 @@ int wc_AesCmacVerify(const byte* check, word32 checkSz,
     return ret;
 }
 
+#ifdef WOLFSSL_SE050
+/**
+ * Use software crypto instead of hardware cryptography. Only supported on
+ * NXP SE050 currently.
+ *
+ * cmac     pointer to Cmac struct, before call to wc_InitCmac()
+ * enabled  0 to use hardware crypto, 1 to use software crypto
+ *
+ * Returns 0 on success, negative on error
+ */
+int wc_CmacUseSoftware(Cmac* cmac, byte enabled)
```

Comment:
Can we just use the `devId` to gate this? Init with `INVALID_DEVID` use software or != INVALID_DEVID to use SE050 HW?

Reply:
Same logic as above.  I could switch back and have HostCrypto code just set "cmac->useSWCrypt = 1" directly.  Then could remove this API.